### ISSUE TITLE
Missing Menu Name label breaks WordPress admin menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#33]
```
What steps will reproduce the problem?
1. Create or edit a custom post type.
2. Navigate to Labels.
3. Leave the Menu Name label blank.
4. Save changes.

What is the expected output? What do you see instead?

I expect to see the name of the custom post type in the WordPress admin menu 
(left sidebar). The Custom Post Type input prompt reads, "Defaults to value of 
name."

However, no menu label text is supplied to the WordPress admin menu and it 
breaks cosmetically, affecting that post type and the menu item below it.


Does the problem continue if you disable all other plugins?
Yes.

What version of the plugin are you using?
Version 0.8.7

What version of WordPress?
3.1

Please provide any additional information below (e.g. Browser, version,
OS):
The WordPress admin menu breakage looks the same in both Firefox 3.6.8 and 
Internet Explorer 8.

```

Original issue reported on code.google.com by `paul.nov...@gmail.com` on 1 Mar 2011 at 7:50

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this!  I'll get a fix implemented in version 0.8.8, due 
out later this week.
```

Original comment by `fireproofsocks` on 1 Mar 2011 at 8:53
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I've got this fixed in my code... still got some more things to fix before 
I can release it, but it's coming.
```

Original comment by `fireproofsocks` on 3 Mar 2011 at 5:15
- Changed state: **Fixed**
